dt/uni05epsilon/edpm/nodeset/nova_custom.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nova-custom-config
+data:
+  25-nova-custom.conf: |
+    # NOTE(gibi): We need to disable multipath as IPv6 is not fully
+    # working in this job with NetApp.
+    # Enable multipath when OSPRH-7393 is resolved.
+    [libvirt]
+    volume_use_multipath = False
+
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: nova-custom
+spec:
+  label: dataplane-deployment-nova-custom
+  configMaps:
+    - nova-custom-config
```

Comment:
as a side note because you hae used this name
i belive cifmw_edpm_deploy_nova_compute_extra_config will also allow job to override the default config map you are specifying above

Reply:
I'm not sure as that requires nova-custom service to be used but this job explicitly used the default nova service in the nodeset

Reply:
right 

the ci framework generates

nova-extra-config

so if you use that name for the configmap instead
then we will be able to use cifmw_edpm_deploy_nova_compute_extra_config

in the ci job to modify this if needed in the future.

it thought you had used the same name but you have not.

Reply:
supporting that is not strictly required but if you do respin using the same name will make things more consistent across the jobs

Reply:
maybe some missing context the ci framework always creates the 

nova-extra-config configmap even if you don't specify
cifmw_edpm_deploy_nova_compute_extra_config
its just empty in that case.

so you don't actlly need to use nova-custom for this to work you just need a custom dataplaneservice for nova that references nova-extra-config to be able to customise vai cifmw_edpm_deploy_nova_compute_extra_config

Reply:
I don't want to create a dependency here to the ci-framework behavior as the architecture repo is expected to be used without ci-framework too.

Reply:
this would not create a depency im not suggesting you remove the default you  are creating

anyway we can keep the divreggance i just mentioned it because it was easy to make both ways fo customisting this work
we don't actually need to respine for that

Reply:
wouldn't ci-framework override the CM content if I use the same CM name?